crates/lib3h/src/engine/ghost_engine.rs

```diff
@@ -811,6 +780,54 @@ fn includes(list_a: &[Address], list_b: &[Address]) -> bool {
     set_b.is_subset(&set_a)
 }
 
+pub fn handle_gossip_to<
```

Comment:
Is this all new code or ported/moved from somewhere?

Reply:
moved in from real.